_pins/shobhit1.json

```diff
@@ -0,0 +1,5 @@
+---
+githubHandle: "shobhit1"
```

Comment:
@devesh3710 great work on adding your pin - one small change, you will need to remove the quote marks `"` from your pin. After you have done that, I will be able to merge your pull request!

Reply:
@Shobhit1 I guess this pertains to your pull request.